core/src/packets/icmp/v6/ndp/router_advert.rs

```diff
@@ -46,93 +46,54 @@ const O_FLAG: u8 = 0b0100_0000;
-///                                 message fields or options. Options that need
-///                                 time limits for their information include
-///                                 their own lifetime fields.
+/// - *Router Lifetime*:
+///                     16-bit unsigned integer.  The lifetime associated
+///                     with the default router in units of seconds.
 ///
-/// - *Reachable Time*:             32-bit unsigned integer. The time, in
-///                                 milliseconds, that a node assumes a neighbor
-///                                 is reachable after having received a
-///                                 reachability confirmation. Used by the
-///                                 Neighbor Unreachability Detection algorithm
-///                                 (see Section 7.3).  A value of zero means
-///                                 unspecified (by this router).
+/// - *Reachable Time*: 32-bit unsigned integer.  The time, in
+///                     milliseconds, that a node assumes a neighbor is
+///                     reachable after having received a reachability
+///                     confirmation.
 ///
-/// - *Retrans Timer*:              32-bit unsigned integer.  The time, in
-///                                 milliseconds, between retransmitted Neighbor
-///                                 Solicitation messages.  Used by address
-///                                 resolution and the Neighbor Unreachability
-///                                 Detection algorithm (see Sections 7.2
-///                                 and 7.3).  A value of zero means unspecified
-///                                 (by this router).
+/// - *Retrans Timer*:  32-bit unsigned integer.  The time, in
+///                     milliseconds, between retransmitted Neighbor
+///                     Solicitation messages.
 ///
 /// Possible options:
 ///
-/// - *Source link-layer address*:  The link-layer address of the interface from
-///                                 which the Router Advertisement is sent. Only
-///                                 used on link layers that have addresses. A
-///                                 router MAY omit this option in order to
-///                                 enable inbound load sharing across multiple
-///                                 link-layer addresses.
+/// - *Source link-layer address*:
+///                     The link-layer address of the interface from which
+///                     the Router Advertisement is sent.
 ///
-/// - *MTU*:                        *SHOULD* be sent on links that have a
-///                                 variable MTU (as specified in the document
-///                                 that describes how to run IP over the
-///                                 particular link type). *MAY* be sent on
-///                                 other links.
+/// - *MTU*:            SHOULD be sent on links that have a variable MTU
```

Comment:
it's all spacing. if the label is too long rather than widen the line with more tabs, they just start the description on the new line to keep the width more compact.

Reply:
I get that, but all the other options start on the next line, but not this one. Just popped out in review is all.

Reply:
right, 👍 I maintained the line breaks and spacing from the rfc.